### ISSUE TITLE
[Game Crash] Crash on song end

### ISSUE THREAD

**Issue by TheLastProject** [TeamRizu/OutFox#351]
**Describe how it crashed by giving us those informations:**

1. What you were doing before it crashed and what action made the game to crash.
2. Upload the crashinfo.txt (GameFolder/crashinfo.txt)
3. Upload the info.txt and log.txt (GameFolder/Logs)

**Explain how we can reproduce the error**

1. Start the game
2. Pick any song
3. Finish the song
4. Game crashed

**Sreenshots/Videos**

If possible, take screenshots or record the crash happening.

**Additional information**

Could not find any `crashinfo.txt` file:
```
pi@raspberrypi:~ $ find . -iname "crashinfo.txt"
pi@raspberrypi:~ $ 
```
[log.txt](https://github.com/TeamRizu/OutFox/files/6644068/log.txt)
[info.txt](https://github.com/TeamRizu/OutFox/files/6644069/info.txt)

**Comment by Scraticus** [comment]
Hiya!

Which raspberry pi model is this running on?

Are you playing via headphones, or HDMI?
You also need to swap your ```VideoRenderers=glad``` to ```VideoRenderers=opengl``` as it's more stable on the pi, and i see you are on a MESA build that is under 21 (You will get much more performance on 20+ as it adds better support)

Let me know and we can try to resolve your issues.

**Comment by TheLastProject** [comment]
It's a Raspberry Pi 4B and I'm playing via HDMI:

```
pi@raspberrypi:~ $ uname -a
Linux raspberrypi 5.10.17-v8+ #1421 SMP PREEMPT Thu May 27 14:01:37 BST 2021 aarch64 GNU/Linux
pi@raspberrypi:~ $ cat /sys/firmware/devicetree/base/model
Raspberry Pi 4 Model B Rev 1.2
```

Setting `VideoRenderers=opengl` in `Save/Preferences.ini` indeed fixed the crash (but setting it through "Experimental Settings" did not, it seemed to have decided to fall back to GLAD anyway.

I have attached the logs of the attempt where I set it using the `Experimental Settings` menu, given that the fallback behaviour may be an error. Just in case it may help.
[info.txt](https://github.com/TeamRizu/OutFox/files/6644140/info.txt)
[log.txt](https://github.com/TeamRizu/OutFox/files/6644141/log.txt)

Anyway, I can play now, so I'm happy, thank you! Should I close this issue or do you want to leave it open? If there is any way I can help provide more info to make the game behave better (maybe just error instead of crash or whatever) I'll happily do so.

**Comment by Scraticus** [comment]
Thanks for letting me know! 

GLAD shouldn't be set on the Raspberry Pi, so it is indeed a bug you have spotted. I will leave this open so it reminds me to fix this, and so folks who search for the problem can find it.

Thanks again for taking the time to report this!

**Comment by Scraticus** [comment]
This is confirmed fixed in 4.9.9, we've changed the way the initialisation happens, so it shouldn't break again.
